templates/reset_password/form.html.twig

```diff
@@ -0,0 +1,26 @@
+{% extends 'base.html.twig' %}
+
+{% block title %}Hello {{ email }}!{% endblock %}
+
+{% block body %}
+    <style>
```

Comment:
You've created two twigs for each form. Please note that you can use the same twig. Both ways are fine but you should be aware of both. Give it a try and see how using a single twig file per form would look like